cel/options.go

```diff
@@ -126,9 +126,63 @@ func Macros(macros ...parser.Macro) EnvOption {
 // If all references within an expression are relative to a protocol buffer package, then
 // specifying a container of `google.type` would make it possible to write expressions such as
 // `Expr{expression: 'a < b'}` instead of having to write `google.type.Expr{...}`.
-func Container(pkg string) EnvOption {
+func Container(name string) EnvOption {
 	return func(e *Env) (*Env, error) {
-		e.pkg = packages.NewPackage(pkg)
+		cont, err := e.Container.Extend(containers.Name(name))
+		if err != nil {
+			return nil, err
+		}
+		e.Container = cont
+		return e, nil
+	}
+}
+
+// Abbrevs configures a set of simple names as abbreviations for fully-qualified names.
+//
+// An abbreviation (abbrev for short) can be useful when working with variables, functions, and
+// especially types from multiple namespaces:
+//
+//    // CEL object construction
+//    qual.pkg.version.ObjTypeName{
+//       field: alt.container.ver.FieldTypeName{value: ...}
+//    }
+//
+// Only one qualified path may be used as CEL container, so at least one of these references is
+// a long qualified name within an otherwise short CEL program. Using the following abbreviations,
+// the program becomes much simpler:
+//
+//    // CEL Go option
+//    Abbrevs("qual.pkg.version.ObjTypeName", "alt.container.ver.FieldTypeName")
+//    // Simplified Object construction
+//    ObjTypeName{field: FieldTypeName{value: ...}}
+//
+// There are a few rules for the qualified names and the simple name abbreviations generated from
+// them:
+// - Qualified names must be dot-delimited, e.g. `package.subpkg.name`.
+// - The last element in the qualified name is the simple name used as an abbreviation.
+// - Abbreviations must not collide with each other.
+// - The abbreviation must not collide with the root-level container name.
```

Comment:
What is a "root-level container name"?  Is it an unqualfied name?

Reply:
I think I mean, the first component of a qualified container name.